src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -113,10 +116,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
 
     /**
      * Loads data from this storage file.
-     *
+     * @throws FileNotFoundException if the file got deleted.
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, FileNotFoundException {
```

Comment:
As discussed in the tutorial, load() method is not the best place  to check for the exception. It is best to check it in the save()

Purpose is to check accidental file deletes and not check missing file initially.